### ISSUE TITLE
Footer Links When hovered get's underlined to focus on Link

### ISSUE THREAD

**Issue by Gurmeet-Singh657** [Stoccoin-Official/Stoccoin-Website#186]
Hye
I want to add the such that Footer Link when hovered gets underlined !!
Plz Assign me this issue as GSSOC'23 Contributor !!

Like This 🔽
![image](https://github.com/Stoccoin-Official/Stoccoin-Website/assets/83361657/0feb2b09-f193-4cc3-9538-12251f638d39)

**Comment by Gurmeet-Singh657** [comment]
@royalpinto007  Waiting !!

**Comment by royalpinto007** [comment]
Yeah, cool. Please go ahead and submit a PR. You need not wait :)

**Comment by sagnik-p** [comment]
This issue has been inactive for a long time
I would like to work on this issue
Please assign

**Comment by Gurmeet-Singh657** [comment]
The issue has been completed now @sagnik-p . Just merging the pull request is left !!
